src/ir/proto/raksha_ir.proto

```diff
@@ -126,3 +126,163 @@ enum AnalysisResultStatus {
 message AnalysisResult {
   AnalysisResultStatus result = 1;
 }
+
+message TypeProto {
```

Comment:
Can we split this off to a separate proto file  (e.g., type.proto) instead of adding this to raksha_ir.proto?